oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -80,7 +80,7 @@ public void testOperatingSystem() {
         assertNotNull(proc.getGroupID());
         assertNotNull(proc.getState());
         assertEquals(proc.getProcessID(), os.getProcessId());
-        assertTrue(proc.getParentProcessID() >= 0);
+        os.getProcess(proc.getParentProcessID());
```

Comment:
This test as you've written it introduces a race condition, as the parent may have terminated by the time you get to this test, and I'm actually surprised it didn't fail the CI tests. 

Given that this is a snapshot of the process state at the time you ran the query, you should be able to test the membership of the parent in the set of process IDs returned.